packages/package-json-lint/tests/jest/rules/require-no-terra-base-peer-dependency-versions.test.js

```diff
@@ -6,6 +6,7 @@ describe('require-no-terra-base-peer-dependency-versions', () => {
       ruleConfig: {
         severity: 'error',
       },
+      type: 'module',
```

Comment:
Should this also need to be updated to projectType?

Reply:
Updated here https://github.com/cerner/terra-toolkit/pull/700/commits/25a0615a2844da242fd6dbba5a9852e03b5e402b.